.github/workflows/test-and-push.yml

```diff
@@ -0,0 +1,49 @@
+name: Test ddocs and push to builds server
+
+on: [push, pull_request]
+
+env:
+  BUILDS_COUCH_URL: ${{ secrets.BUILDS_URL }}
+
+jobs:
+
+  test:
+    name: Test
+    runs-on: ubuntu-18.04
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v1
+        with:
+         node-version: 16.x
+
+      - name: Run CouchDb in Docker
+        run: |
+          docker run -d -p 5984:5984 --name couch couchdb:2.3.1
+          until nc -z localhost 5984; do sleep 1; done
```

Comment:
TIL we can use the Docker CLI and utilities like netcat in GitHub Actions :sparkles: